test/backstage.test.ts

```diff
@@ -0,0 +1,62 @@
+import * as cdk from 'aws-cdk-lib';
```

Comment:
Can we have at least one test that succeeds? I see a bunch of tests that cover failure scenarios. Those are good, but proving that happy path works is the primary objective provided the addon is not part of our e2e tests.

Reply:
added